javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,163 @@
+// Task 1: Add two parameter in function.
+
+function getFullName(firstname, lastname) {
+    console.log(`"${firstname} ${lastname}"`);
+}
+getFullName("Benjamin", "Hughes");
+
+
+// fullname with formalname and gender condition check , also checked empty string in firstname and lastname
+
+const fullName1 = getFullName("Prakash", "Patel", "male", true);
+const fullName2 = getFullName("Smita", "Patel", "female", true);
+const fullName3 = getFullName("Maahit", "Patel", "male", false);
+const fullName4 = getFullName("Preshu", "Patel", "female", true);
+const fullName5 = getFullName("", "", "female", true);
+const fullName6 = getFullName("Prakash", "Patel", "male", true);
+
+function getFullName(firstname, lastname, gender, useFormalName) {
+    if (firstname === "" || lastname === "") {
+        console.warn("please enter first name or last name")
+    }
+    else {
+        if (useFormalName == true) {
+            if (gender == "male") {
+                console.log(`"Lord ${firstname} ${lastname}"`);
+            }
+            else
+                console.log(`"${firstname} ${lastname}"`)
+        } else {
+            console.log(`"${firstname} ${lastname}"`);
+        }
+    }
+}
+getFullName(fullName1);
+
+
+// Task 2 Eventday calculator 
+
+const weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const weekDay = new Date();
+
+function getEventWeekday(eventday) {
+
+    weekDay.setDate(weekDay.getDate() + eventday);
+    return weekDays[weekDay.getDay()];
+}
+console.log(`Event held on ${getEventWeekday(6)}`);
```

Comment:
getEventWeekday(1);
If I call several time result show me same every time but I can't understand reason behind call several time.

Reply:
Now will work as it should because you refactored the function and put `const weekDay = new Date();`  inside the function 🙂 
I wanted to help you to solve the issue on your own, trying to understand why it should be inside the function.

Reply:
Thanks , I understand your point.